src/cli.js

```diff
@@ -118,8 +118,13 @@ export default function cli(args, log, error, exit) {
       skipMissing: opt.argv.skipMissing,
     }))
     .then(result => print(result, log, opt.argv.json))
-    .then(({ dependencies: deps, devDependencies: devDeps }) =>
-      exit(opt.argv.json || (deps.length === 0 && devDeps.length) === 0 ? 0 : -1))
+    .then(({ dependencies: deps, devDependencies: devDeps, missing }) => {
+      const hasNoErrors = deps.length === 0 &&
```

Comment:
Could you please make the variable `hasError`? A negative expression seems a little bit...

Reply:
I called it `hasErrors` since there can be multiple errors still